changelog/17650.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+cli: Add patch support for non-KVv2 endpoints
```

Comment:
I'm not sure if we want to be more specific that this will provide patch support for endpoints that already support HTTP PATCH. Users might interpret this entry to mean that patch support has been broadly introduced.

Reply:
Let me know if this iteration is better?

Reply:
Looks good to me, thanks!